docs/Measuring_Resonances.md

```diff
@@ -228,7 +307,9 @@ Recv: // adxl345 values (x, y, z): 470.719200, 941.438400, 9728.196800
 ```
 
 If you get an error like `Invalid adxl345 id (got xx vs e5)`, where `xx`
-is some other ID, it is indicative of the connection problem with ADXL345,
+is some other ID, immediately try again. There's an issue with SPI
```

Comment:
AFAIK the first measurement with a Pico will always fail. Maybe it should be in the section of the Pico itself

Reply:
Indeed but it wasn't always the case:  https://github.com/Klipper3d/klipper/issues/4712

Reply:
I believe it's a bug as pointed out by joaobarros. I don't believe it's Pico-specific. For that reason, I don't believe it should be in the Pico section. Strong opinion, loosely held.

Reply:
Well, if a bug or not: I only see this behavior with the Pico. Not on a RPi 3 and also not on a Orange Pi 3 LTS

Reply:
Fascinating. After this PR, I may take a stab at fixing the bug. I don't see how it would be Pico-specific or Pi 4 specific but maybe I'll find out.

Reply:
It's specifically a bug in adxl345.py. A function to initialize the chip was removed in commit https://github.com/Klipper3d/klipper/commit/8cfaed441d1ec295c8c4be8e0733099fe3f3395a
I don't pretend to understand the reasoning behind the change, but that is the change that introduced the "always fails the first time" bug.

Reply:
Thanks, glibifrons. That at least should make it easy to reproduce and examine, whether that's done by me or by someone else. :)

Reply:
@glabifrons could you open a ticket for this? It's an annoying bug that should be fixed if possible, especially if it's simply reverting part of a commit.

Reply:
@thijstriemstra I did, over a year ago: https://github.com/Klipper3d/klipper/issues/4712